### PR TITLE
adjust redis connector to point at correct db-index

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

File: Containers/nextcloud/Dockerfile [hunk 1]
```diff
@@ -131,7 +131,7 @@ RUN set -ex; \
     \
     { \
         echo 'session.save_handler = redis'; \
-        echo 'session.save_path = "tcp://${REDIS_HOST}:6379?auth=${REDIS_HOST_PASSWORD}"'; \
+        echo 'session.save_path = "tcp://${REDIS_HOST}/0:6379?auth=${REDIS_HOST_PASSWORD}"'; \
```
Comment (docjyJ) [inline Containers/nextcloud/Dockerfile:134]: It doesn't respect the format of a URI, if it's wanted it's strange...
```suggestion
        echo 'session.save_path = "tcp://${REDIS_HOST}:6379/0?auth=${REDIS_HOST_PASSWORD}"'; \
```

Comment (szaimen) [reply Containers/nextcloud/Dockerfile:134]: Yes, you are right. Made a typo here...

File: Containers/notify-push/start.sh [hunk 2]
```diff
@@ -50,7 +50,7 @@ if [ -z "$POSTGRES_PORT" ]; then
 fi
 # Set sensitive values as env
 export DATABASE_URL="postgres://oc_$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB"
-export REDIS_URL="redis://:$REDIS_HOST_PASSWORD@$REDIS_HOST"
+export REDIS_URL="redis://:$REDIS_HOST_PASSWORD@$REDIS_HOST/0"
```
Comment (szaimen) [inline Containers/notify-push/start.sh:53]: todo: read varibable

**Comment by szaimen** [comment]
This is now released with 9.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Pull request merged**